jira-environment-ticket-create/src/index.ts

```diff
@@ -12,8 +12,29 @@ async function run() {
     const issueTypeId = core.getInput('issueTypeId', { required: true });
     const nameField = core.getInput('nameField', { required: true });
     const urlField = core.getInput('urlField', { required: true });
+    const projectKey = core.getInput('projectKey', { required: true });
+    const jiraEnvironmentField = core.getInput('jiraEnvironmentField', { required: true });
 
     const jiraBase64Credentials = Buffer.from(`${jiraUsername}:${jiraPassword}`).toString('base64');
+    const environmentJql = `project = ${projectKey} AND "${jiraEnvironmentField}" ~ "${environmentName}"`;
+
+    core.info('Checking if issue already exists');
+    const existingUrl = encodeURI(`https://${jiraServer}/rest/api/latest/search?jql=${environmentJql}&fields=labels`);
```

Comment:
Guessing this isn't needed?
```suggestion
    const existingUrl = encodeURI(`https://${jiraServer}/rest/api/latest/search?jql=${environmentJql}`);
```